[BUG]: version 5.11.0, ext tree is unusable (PHP 8.5)

Building from `ext` tree is broken

```
/builddir/build/BUILD/php-phalcon5-5.11.0/cphalcon-5.11.0/ext/kernel/backtrace.c:18:10: fatal error: ext/standard/php_smart_string.h: No such file or directory
 #include <ext/standard/php_smart_string.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

```

Building from `build/phalcon` is OK

So there is something wrong in the rpository.


I know, building from  `build/phalcon` is the recommended way
But building from  `ext` is much faster, especially when you have a lot of core (4' vs 8' on aarch64 using only 24 cores)

--- a/ext/kernel/backtrace.c
+++ b/ext/kernel/backtrace.c
@@ -12,6 +12,7 @@
 #ifndef ZEPHIR_RELEASE
 #if defined(linux) && !defined(ALPINE_LINUX) || defined(DARWIN) || defined(__APPLE__)
 
+#include <php.h>
 #include <execinfo.h>
 #include <Zend/zend.h>
 #if PHP_VERSION_ID < 80500

